### PR TITLE
Make InvalidCommandException handleable by CLI.exceptionHandler

### PR DESCRIPTION
**Pull request by jwang47**

### CODE REVIEW

**Comment by awholegunch** [comment]
LGTM.

File: common-cli/src/main/java/co/cask/common/cli/CLI.java [hunk 1]
```diff
@@ -127,8 +127,8 @@ public ConsoleReader getReader() {
    * @param output the {@link PrintStream} to write messages to
    */
   public void execute(String input, PrintStream output) throws InvalidCommandException {
```
Comment (bdmogal) [inline common-cli/src/main/java/co/cask/common/cli/CLI.java]: Checking again, but this method should probably not throw ICE if it is handled?

Comment (bdmogal) [inline common-cli/src/main/java/co/cask/common/cli/CLI.java]: Yeah, shouldn't throw anymore.

Comment (jwang47) [inline common-cli/src/main/java/co/cask/common/cli/CLI.java]: Thanks for the catch.

**Pull request merged**